pool/app/mailing/entity.ml

```diff
@@ -281,5 +282,10 @@ let default_sort =
 ;;
 
 let default_query =
-  Query.{ pagination = None; search = None; sort = Some default_sort }
+  Query.
```

Comment:
This record creation appears many times, use the existing `create ~sort:default_sort ()` function